util/topgen/c.py

```diff
@@ -52,13 +52,20 @@ def __init__(self, name, base_addr, size_bytes):
         self.name = name
         self.base_addr = base_addr
         self.size_bytes = size_bytes
+        self.size_words = int(str(size_bytes), 0) // 4
```

Comment:
I'm not keen on the casts here. Do we ever pass in something other than a string? If so, I'd propose fixing the call site (so that `self.size_bytes` has a predictable type).

Also, what should happen if `size_bytes == 7`? This code rounds down, which looks surprising to me.

Also, do you actually use this field anywhere? If not, maybe there's no need to add it?

Reply:
Hmmm, I think you are right, I am not using it, silly :/

Reply:
Okay, I actually do use it in the template:
`#define ${region.size_words_name().as_c_define()} ${region.size_words}u` :)

Reply:
Cool. Probably still needs rounding up, though.

Reply:
Indeed, thank you for catching that!

Reply:
Done.